Start adding AddressPageShown Event

moiikana opened the pull request:
- still not working properly on mobile VAR

wiese commented on desktop/css/styles.pcss:
```diff
@@ -68,9 +69,9 @@
-		font-family: serif;
-		font-size: 1.2em;
-		line-height: 1.2em;
+		font-family: 'Source Sans', 'Linux Libertine', 'Linux Libertine O', 'Georgia', 'Bitstream Charter', serif;
```
Interesting mix of sans serif and serif fonts. Intentional?

KaiNissen replied:
No, not intentional. Good catch!

gbirke committed:
Implement tracking architecture

Entry points pass mapping factories to Tracker, Tracker classes convert
Domain events to specific tracker events. The list of allowed events
determines which events are actually sent.

We'll have to see how we can support sending the dynamic banner size to
the tracker, maybe we'll refactor that.

Co-Authored-By: Corinna Hillebrand <corinna.hillebrand@wikimedia.de>
Co-Authored-By: Sperling-0 <tanuja.doriya@wikimedia.de>

gbirke committed:
Fix CI for WPDE banners

gbirke committed:
Add Expand mini banner event and start with already donated event

Co-Authored-By: Corinna Hillebrand <corinna.hillebrand@wikimedia.de>
Co-Authored-By: Sperling-0 <tanuja.doriya@wikimedia.de>

gbirke committed:
Add UpgradeToYearlyFormPageShownEvent tracking

Rename FormControllerCtrl to FormController where banners share a
FormController implementation.

Add tests for FormControllers

gbirke committed:
Add CustomAmountFormPageShownEvent

Co-Authored-By: Corinna Hillebrand <corinna.hillebrand@wikimedia.de>
Co-Authored-By: Sperling-0 <tanuja.doriya@wikimedia.de>

gbirke committed:
Add increase and decrease custom amount events

Co-Authored-By: Corinna Hillebrand <corinna.hillebrand@wikimedia.de>
Co-Authored-By: Sperling-0 <tanuja.doriya@wikimedia.de>

moiikana committed:
Add decrease and increase event to all channels

moiikana committed:
Start adding AddressPageShown Event

- still not working properly on mobile VAR
